features/step_definitions/cco.rb

```diff
@@ -0,0 +1,10 @@
+Given /^save of custom resource `(.+?)` to file>(.+)$/ do |what,filepath|
```

Comment:
Since most of our current steps have a *subject* in the sentence, Do you mind updating the step to `/^I save the custom resource `(.+?)` to file>(.+)$/` (or `Admin save .... to`)

Reply:
sure, wait for cucushift case review, then i will fix the both

Reply:
Good approach. Thanks.

The step is mainly for CCO currently, but it's a common step. Would you mind put it into `features/step_definitions/common.rb` ?

Reply:
sure